src/main/java/xyz/nucleoid/extras/lobby/item/TaterBoxItem.java

```diff
@@ -148,13 +156,13 @@ private MutableText getTitle(ItemStack stack) {
         return result;
     }
 
-    private GuiElement createGuiElement(ItemStack stack, PlayerEntity user, Hand hand, ItemConvertible icon, Text name, Identifier taterId) {
+    private GuiElement createGuiElement(ItemStack stack, PlayerEntity user, Hand hand, ItemConvertible icon, Text name, Identifier taterId, boolean selectable) {
```

Comment:
Since `taterId` is not applicable when `selectable` is `false`, this method should be split.